docs/architecture/deployment/installation-guide.md

```diff
@@ -36,38 +36,105 @@ To install Prophecy, the following infrastructure requirements must be satisfied
+| `postgres.secretName`                      | string                   | Specifies the postgres AWS secret manager name from where postgres credentials are to be fetched from.                                                                                                                           |                                     |
+| `postgres.secretLocation`                  | string                   | Specifies the postgres AWS secret manager region from where postgres credentials are to be fetched from.                                                                                                                         |                                     |
+| `postgres.volume`                          | -                        |                                                                                                                                                                                                                                  |                                     |
+| `postgres.volume.requests`                 | string                   | Specifies the volume size of the PVC used by postgres                                                                                                                                                                            | `25Gi`                              |
+| **PLATFORM**                               |
+| `platform.enabled`                         | bool                     | Specifies if one/more of the platform components are to be enabled                                                                                                                                                               | `true`                              |
+| `platform.elasticsearch.enabled`           | bool                     | Specifies if elasticsearch is to be enabled and deployed for supporting Prophecy search feature                                                                                                                                  | `true`                              |
+| `platform.ingressNginx.enabled`            | bool                     | Specifies if nginx controller is to be deployed for ingress routing                                                                                                                                                              | `true`                              |
+| `platform.loki.enabled`                    | bool                     | Specifies if loki is to be enabled for log collection                                                                                                                                                                            | `false`                             |
```

Comment:
Loki/promtail/prometheus options we can remove , wil make it by default we dont install